system/handlers/admin/SiteTree.cfc

```diff
@@ -264,6 +264,7 @@ component extends="preside.system.base.AdminHandler" {
 		prc.page         = _getPageAndThrowOnMissing( argumentCollection=arguments, allowVersions=true );
 		prc.canPublish   = _checkPermissions( argumentCollection=arguments, key="publish", pageId=pageId, throwOnError=false );
 		prc.canSaveDraft = _checkPermissions( argumentCollection=arguments, key="saveDraft", pageId=pageId, throwOnError=false );
+		rc._backToEdit   = cookie.sitetree_editPage_backToEdit ?: false;
```

Comment:
I'd suggest the following here is more robust:

```
rc._backToEdit = IsTrue( cookie.sitetree_editPage_backToEdit ?: "" );
```

`IsTrue()` is a preside helper function for evaluating truthy values.

Reply:
Also, please use `cookieService` to get / set cookie variables:

```
rc._backToEdit = IsTrue( cookieService.getVar( "sitetree_editPage_backToEdit", "" ) );

// ...

cookieService.setVar( name="sitetree_editPage_backToEdit", value=true );
```